staging/src/k8s.io/kubectl/pkg/cmd/top/top_pod.go

```diff
@@ -282,7 +282,7 @@ func checkPodAge(pod *corev1.Pod) error {
 		message := fmt.Sprintf("Metrics not available for pod %s/%s, age: %s", pod.Namespace, pod.Name, age.String())
 		return errors.New(message)
 	} else {
-		klog.V(2).Infof("Metrics not yet available for pod %s/%s, age: %s", pod.Namespace, pod.Name, age.String())
+		klog.V(2).InfoS("Metrics not yet available for pod", "pod.Namespace", pod.Namespace, "pod.Name", pod.Name, "age", age.String())
```

Comment:
Is this a user-visible change? If it is, we should update the changelog entry from `NONE`.

Reply:
Done. :D